api-reference/docstubs/api/group-post-members.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Create members"
+description: "Create members by posting to the members collection."
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# Create members
+
+Namespace: microsoft.graph
+
+Create members by posting to the members collection.
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/concepts/permissions-reference.md).
+
+|Permission type|Permissions (from most to least privileged)|
+|:---|:---|
+|Delegated (work or school account)|**TODO: Determine scopes **|
+|Delegated (personal Microsoft account)|Not supported.|
+|Application|**TODO: Determine AppOnly scopes **|
+
+## HTTP request
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+POST /groups/{groupsId}/members/$ref
+POST /me/joinedGroups/{groupId}/members/$ref
+```
+
+## Request headers
+|Name|Description|
+|:---|:---|
+|Authorization|Bearer {token}.Required|
+|Content-Type|application/json.Required|
+
+## Request body
+In the request body, supply a JSON representation for the [directoryObject](../resources/directoryobject.md) object.
+
+The following table shows the properties that are required when you create the [directoryObject](../resources/directoryobject.md).
+
+|Property|Type|Description|
+|:---|:---|:---|
+|id|String| Inherited from [entity](../resources/entity.md)|
+|deletedDateTime|DateTimeOffset||
+
+
+
+## Response
+If successful, this method returns a `201 Created` response code and a [directoryObject](../resources/directoryobject.md) object in the response body.
```

Comment:
This is wrong.  It's a 204 No Content.
Again - navigation and contained navigation is being conflated.